docs/openapi-backend.yaml

```diff
@@ -0,0 +1,331 @@
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateSignatureBody"
+      responses:
+        "201":
+          description: Signature created
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+
+components:
+  securitySchemes:
+    Bearer:
+      type: apiKey
+      name: Authorization
+      in: header
+  responses:
+    NotFound:
+      $ref: "https://raw.githubusercontent.com/pagopa/io-functions-sign/v1.0.0-RELEASE/openapi-issuer.yaml#/components/responses/NotFound"
+    BadRequest:
+      $ref: "https://raw.githubusercontent.com/pagopa/io-functions-sign/v1.0.0-RELEASE/openapi-issuer.yaml#/components/responses/BadRequest"
+    Forbidden:
+      $ref: "https://raw.githubusercontent.com/pagopa/io-functions-sign/v1.0.0-RELEASE/openapi-issuer.yaml#/components/responses/Forbidden"
+    Unexpected:
+      $ref: "https://raw.githubusercontent.com/pagopa/io-functions-sign/v1.0.0-RELEASE/openapi-issuer.yaml#/components/responses/Unexpected"
+  schemas:
+    Id:
+      type: string
+      description: Entity Id
+      format: ulid
+      example: 01ARZ3NDEKTSV4RRFFQ69G5FAV
+
+    Timestamp:
+      type: string
+      format: UTCISODateFromString
+      description: A date-time field in ISO-8601 format and UTC timezone.
+      x-import: "@pagopa/ts-commons/lib/dates"
+      example: "2018-10-13T00:00:00.000Z"
+
+    Document:
+      type: object
+      properties:
+        id:
+          $ref: "#/components/schemas/Id"
+        metadata:
+          $ref: "#/components/schemas/DocumentMetadata"
+        created_at:
+          $ref: "#/components/schemas/Timestamp"
+        updated_at:
+          $ref: "#/components/schemas/Timestamp"
+      required:
+        - id
+        - status
+        - metadata
+        - created_at
+        - updated_at
+
+    DocumentMetadata:
+      type: object
+      properties:
+        title:
+          type: string
+        signature_fields:
+          type: array
+          items:
+            $ref: "#/components/schemas/SignatureField"
+
+    SignatureFieldAttributes:
+      type: object
+      properties:
+        unique_name:
+          type: string
+
+    SignatureFieldToBeCreatedAttributes:
+      type: object
+      properties:
+        coordinates:
+          type: object
+          properties:
+            x: { type: number, minimum: 0 }
+            y: { type: number, minimum: 0 }
+        page:
+          type: number
+          minimum: 0
+        size:
+          type: object
+          properties:
+            w: { type: number, minimum: 0 }
+            h: { type: number, minimum: 0 }
+    Clause:
+      type: object
+      properties:
+        title:
+          type: string
+        type:
+          type: string
+          enum: ["REQUIRED", "UNFAIR", "OPTIONAL"]
```

Comment:
Same thing for the enums. What do you think?

Reply:
Having a one more definition simplifies the use of imports.
`import { SignatureRequestStatusEnum } from "../../../../generated/definitions/fci/SignatureRequestStatus";` instead of
`import { StatusEnum as SignatureRequestStatusEnum } from "../../../../../../definitions/fci/SignatureRequestDetailView";`

Reply:
You are right. I have proceeded to separate the enums into a separate refs!